db/migrate/20200515122239_create_households.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Migration to create the households table
+class CreateHouseholds < ActiveRecord::Migration[6.0]
+  # This class is generated after running "jets db:generate create_households"
+  # The command on its own will generate a migration class that initially,
+  # only creates the table. Additional columns can be added after running
+  # the create command.
+  def change
+    # Implicitly creates an id field for primary key
+    create_table :households do |t|
```

Comment:
I think we need the concept of a user or owner id in this table. Otherwise there will be no way to identify which users are allowed to see and update which households.

Reply:
According to the ERD, that has been added to the household_members table - is_head_of_household. What are your thoughts after knowing that @nvengal ?

Reply:
- is_head_of_household - Yes, that is important - who is the current HOH of the household?

Reply:
Got ya. That seems fine. I think the auth pieces of which users can modify which households will be more clear after the members get added.